Misc/NEWS.d/next/C API/2021-03-15-15-06-16.bpo-43502.G-uhre.rst

```diff
@@ -0,0 +1,2 @@
+Convert ``PyExceptionClass_Check`` from macro to static inline function.
```

Comment:
I'm not sure that it's worth it to document such change. The double evaluation issue exists in Python 2.7 and is likely older. If it's clearly a bugfix, it should be documented and backported. It doesn't seem to be the case here.

Reply:
Removed in fe5b0ff58033f96f1cc7a05598e371266dde0d1d